proto/provider/akash/inventory/v1/resourcepair.proto

```diff
@@ -25,7 +25,6 @@ message ResourcePair {
   ];
   repeated akash.base.v1beta3.Attribute attributes = 3 [
     (gogoproto.nullable)     = true,
```

Comment:
lets make it non-nullable `(gogoproto.nullable) = false` so item in slice are value instead of pointers